devicetypes/smartthings/aeotec-wallmote.src/aeotec-wallmote.groovy

```diff
@@ -178,7 +178,8 @@ private getSupportedButtonValues() {
 private getButtonAttributesMap() {
     if (isEverspring()) {[
             0: "pushed",
-            2: "held",
+            1: "held",
+            2: "down_hold",
```

Comment:
why are you changing what event 2 maps to?

Reply:
Before I handled only 02 scene number and mapped that as "held" and skipped the 01 scene number which in fact is for the "release" action. When holding the button the first event which is coming is with 02 scene number, then the 01 is fired. So, the 02 scene number is executing at first while holding the button and then the 01 - because of that I treated the 02 as "held down" and 01 as "held" - the same as it is applied for the Aeotec Nano One button.

Reply:
"down_hold" is more like an up-down toggle switch that is being held down.

So, on this device, if I press down and release the button immediately, I get a scene 0? And then if I press down and hold, I get a scene 2 and a scene 1? Do I get a scene 1 after the scene 0 if I just push and release? Is there ever an instance where I get a scene 1 but never any of the others? If I push and hold, do I get a scene 0, then after some time a scene 2, and then eventually a scene 1?

What I would do to fix ICP-12927 is to have a check in the code that uses `getButtonAttributesMap` to not generate an event if `value == null`. It seems like that is the appropriate solution.

Reply:
Thank you, Donald. Your suggestion sounds much better. 

And yes, if you press once (short press) you get the scene 0, press down+hold you get scene 2 and 1. You don't get scene 1 after scene 0. You never get scene 1 alone, it is always correlated with scene 2 and appears only after scene 2. So the only case when you get scene 1 is while holding the button. If you short press and then press down and hold you get in sequence: scene 0 (for the short press), then scene 2 and 1 right after (for the holding). 

So do you think that we should apply the same logic to the Aeotec Nano Button? Now it is working like I was trying to do this here on the begging. And I verified that the ICP-12926 is resolved with this fix, so half of the PR was still good ;) Now I can say the same about ICP-12927.